### PR TITLE
Small misstype and broken link fixed in the Arduino.md

### PR DESCRIPTION
**Pull request by ValentiWorkLearning**
### Description of the feature or fix

A clear and concise description of what the bug or new feature is.

### Checkpoints
- [x] Follow the [styling guide](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md)
- [ ] Update [CHANGELOG.md](https://github.com/lvgl/lvgl/blob/master/docs/CHANGELOG.md)
- [x] Update the documentation

### CODE REVIEW

File: docs/get-started/arduino.md [hunk 1]
```diff
@@ -45,9 +45,12 @@ LVGL has its own configuration file called `lv_conf.h`. When LVGL is installed t
 
 ## Initialize LVGL and run an example
 
-Take a look at [LVGL_Arduino.ino](https://github.com/lvgl/lvgl/blob/master/examples/LVGL_Arduino.ino) to see how to initialize LVGL.
+Take a look at [LVGL_Arduino.ino](https://github.com/lvgl/lv_arduino) to see how to initialize LVGL.
+Note, that this repository is obsolete and can be used only as an example of initializing the library with a given display driver.
```
Comment (kisvegabor) [inline docs/get-started/arduino.md:49]: > Note, that this repository is obsolete

What do you mean by **this** repository? Which repository?

Comment (ValentiWorkLearning) [reply docs/get-started/arduino.md:49]: Here I mean https://github.com/lvgl/lv_arduino

Comment (kisvegabor) [reply docs/get-started/arduino.md:49]: Why do we need to mention this repo?

Comment (ValentiWorkLearning) [reply docs/get-started/arduino.md:49]: It seems, that it was previously used. I can change it to the correct repository with examples for Arduino.

Comment (kisvegabor) [reply docs/get-started/arduino.md:49]: IMO the last sentence is no required. A link to the Arduino.ino should be enough to point the users in the right direction.

**Comment by embeddedt** [comment]
In general, the Arduino page was quite outdated with respect to some changes in v8, so I've updated it here based on this PR: https://github.com/lvgl/lvgl/commit/9a77102c40f68140d0ba2c6c5e493e51a8773f64

Please don't hesitate to open another PR if you have further suggestions!

**Pull request closed without merging**